es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+    <b-container class="my-450">
+        <b-col
```

Comment:
This was on the original bootstrap page too and doesn't seem to be a problem , but a `<b-row>` around this `b-col` would be good practice.